### ISSUE TITLE
Nested floating windows dont scroll properly 

### ISSUE THREAD

**Issue by goblinfactory** [goblinfactory/konsole#18]
When nesting a floating window, inside another floating window, the new nested window does not scroll correctly.

```
var w = new Window();
var left = w.SplitLeft("left");
var top = left.SplitTop("top");
var bottom = left.SplitBottom("bottom");
```

In the example above, top and bottom are both nested floating windows. When printing to bottom or top, causing the window to scroll, then the scrolling does not happen correctly.

**Comment by goblinfactory** [comment]
Fixed in latest commit, will be included in the next release.

**Comment by goblinfactory** [comment]
Fixed in release 3.3.0
